subsys/net/lib/fota_download/src/fota_download.c

```diff
@@ -140,12 +165,15 @@ int fota_download_init(fota_download_callback_t client_callback)
 
 	callback = client_callback;
 
-	int err = download_client_init(&dfu, download_client_callback);
+	k_delayed_work_init(&restart_dlc_work, download_with_offset);
+
+	int err = download_client_init(&dlc, download_client_callback);
 
 	if (err != 0) {
 		LOG_ERR("download_client_init error %d", err);
```

Comment:
This can't really fail (only if your params are wrong at compile time); you can remove the error log if you want.

Reply:
Done.